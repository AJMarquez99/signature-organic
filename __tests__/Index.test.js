import React from "react"
import renderer from "react-test-renderer"

import IndexPage from '../src/pages/index.js'


describe("Index Page", ()=> {
  it('The index page should render the exact same way every time', () => {
    const index = renderer.create(<IndexPage/>).toJSON()
    expect(index).toMatchSnapshot()
  })
})
