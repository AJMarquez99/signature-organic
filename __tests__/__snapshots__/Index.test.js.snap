// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<div
  id="homepage-body"
>
  <div
    className="is-row"
    id="homepage-hero"
  >
    <div>
      <div
        className="is-row"
        id="homepage-logo-container"
      >
        <div
          className="is-col"
          id="signature-organic-logo"
        >
          <img
            alt="Signature Organic Logo"
            src="https://imgur.com/rbR6lBj.jpg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="is-col"
    id="homepage-company-description"
  >
    <p
      id="homepage-description-content"
    >
      Founded in 2017, Signature Organic is a family run business focused on bringing you the best organic produce in the twin cities. We deliver free range poultry, fresh organic eggs, and vegetables and salads. Order now to get farm fresh produce at your doorstep!
    </p>
  </div>
  <div
    className="is-row is-shadow-1"
    id="homepage-review-section"
  >
    <h3
      className="is-col is-23"
      id="review-section-title"
    >
      What our customers have to say:
    </h3>
    <blockquote
      className="is-col is-23 homepage-customer-review"
    >
      "They are super professional, offer timely delivery and their products are great. Kudos for doing such a great job!"
    </blockquote>
    <blockquote
      className="is-col is-23 homepage-customer-review"
    >
      "Absolutely loved all the products! Everything from the fresh dairy to the lentils and wheat flour! It's great to find organic and healthy food in Islamabad at such reasonable prices."
    </blockquote>
    <blockquote
      className="is-col is-23 homepage-customer-review"
    >
      "Fully satisfied with the product with such a peace of mind that I am giving healthy food to my daughter :)    Keep it up."
    </blockquote>
  </div>
  <div
    style={
      Object {
        "background": "#B67F42",
        "marginTop": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
          }
        }
      >
        Contact Us
      </h1>
    </div>
  </div>
</div>
`;
